bootstrap.go

```diff
@@ -376,6 +381,13 @@ func upgradeToVer9(s Session) {
 	s.Execute("UPDATE mysql.user SET Trigger_priv='Y'")
 }
 
+func upgradeToVer10(s Session) {
+	_, err := s.Execute("UPDATE mysql.user SET `password` = old_password_upgrade(`password`)")
```

Comment:
if use local function, password can not be updated in one UPDATE statement, we should fetch rows from mysql.user and update each rows.